examples/iot-nirvana/README.md

```diff
@@ -86,20 +102,20 @@ root of the project with the following parameters:
 
 Example:
 
-`run_oncloud.sh my-project my-bucket my-topic my_dataset.my_table`
+`run_oncloud.sh $PROJECT_ID $PROJECT_ID $PUBSUB_TOPIC $BIGQUERY_DATASET.$BIGQUERY_TABLE`
 
 ## Temperature sensor
 
-Copy the JAR package containing the client binaries on Google Cloud Storage in
+Copy the JAR package containing the client binaries to Google Cloud Storage in
 the bucket previously created. Run the following command in the `/client`
 folder:
 
-`gsutil cp target/google-cloud-demo-iot-nirvana-client-jar-with-dependencies.jar gs://[BUCKET_NAME]/client/`
+`gsutil cp target/google-cloud-demo-iot-nirvana-client-jar-with-dependencies.jar gs://$PROJECT_ID/client/`
 
 Check that the JAR file has been correctly copied in the Google Cloud Storage
 bucket with the following command:
 
-`gsutil ls gs://[BUCKET_NAME]/client/google-cloud-demo-iot-nirvana-client-jar-with-dependencies.jar`
+`gsutil ls gs://$PROJECT_ID/client/google-cloud-demo-iot-nirvana-client-jar-with-dependencies.jar`
```

Comment:
Same as before, I would suggest creating a $BUCKET_NAME environment variable and using it here.